### PR TITLE
completed patch

### PR DESCRIPTION
**Pull request by will-maso**

### CODE REVIEW

File: __tests__/news.test.js [hunk 1]
```diff
@@ -49,8 +49,53 @@ describe.only("GET/api/articles/:article_id", () => {
...
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+        expect(result.body.article.votes).toBe(90);
       });
   });
+  test("responds with correct error message", () => {
+    return request(app)
+      .patch("/api/articles/9999")
+      .expect(400)
```
Comment (xmenbteam) [inline __tests__/news.test.js:115]: This should still be a 404 and you need to still send a body. A 400 would be a 'bad request' - how do you perform a bad request with a patch?

File: controllers/articles.controllers.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
-const { fetchArticleById } = require("../models/articles.models");
+const {
+  fetchArticleById,
+  changeArticleById,
+} = require("../models/articles.models");
```
Comment (xmenbteam) [inline controllers/articles.controllers.js:4]: Nice folder structure!

File: models/articles.models.js [hunk 3]
```diff
@@ -11,3 +11,18 @@ exports.fetchArticleById = (article_id) => {
...
 };
+
+exports.changeArticleById = (article_id, inc_votes) => {
+  return db
+    .query(
+      "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *",
+      [inc_votes, article_id]
+    )
+    .then((result) => {
+      if (result.rows.length > 0) {
```
Comment (xmenbteam) [inline models/articles.models.js]: Another cool way to write something like this is `if(result.rows.length)`, as 0 is falsy.

**Review by xmenbteam — CHANGES_REQUESTED** [review]
More tests please!

File: __tests__/news.test.js (hunk 1 above)
Comment (will-maso) [reply __tests__/news.test.js:115]: An incorrect article number ?

**Comment by xmenbteam** [comment]
Spoken about this

**Pull request merged**